examples/server_profile.rb

```diff
@@ -15,10 +15,52 @@
   password: ENV['ONEVIEWSDK_PASSWORD']
 }
 
+# my_server_profile_template = 'spt1'
+# my_server_hardware_type = 'SY 480 Gen9 2'
+# my_server_hardware = '0000A66101, bay 5'
+# my_enclosure_group = 'eg1'
 my_server_profile_template = 'spt1'
-my_server_hardware_type = 'SY 480 Gen9 2'
-my_server_hardware = '0000A66101, bay 5'
-my_enclosure_group = 'eg1'
+my_server_hardware_type = 'BL660c Gen9 1'
+my_server_hardware = 'Encl1, bay 2'
+my_enclosure_group = 'EG_1'
+
+# use this to create server profile with volume attachments using api200 or api300
```

Comment:
I feel like this example is becoming heavily poluted by volume attachments. Not sure if that is just my impression? 
This first part became all about VA variables. Perhaps a separated example is in order for this?